app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  # Attributes
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_couter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  # Callback
+  after_save :update_user_posts_counter
+
+  # Methods
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
```

Comment:
### SUGGESTION 

- [ ] It would be better to make the   update_post_counter method private 👍🏿